modules/tensor_mechanics/include/auxkernels/VectorRankFourAux.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef VECTORRANKFOURAUX_H
+#define VECTORRANKFOURAUX_H
+
+#include "AuxKernel.h"
+#include "RankFourTensor.h"
+
+class VectorRankFourAux;
```

Comment:
I'm not sure what the point of VectorRankFourAux and VectorRankTwoAux are.  We already have RankFourAux and RankTwoAux, so can't a user just use a few of those?  I think i'm missing something here....